website/docs/index.html.markdown

```diff
@@ -17,15 +17,11 @@ Use the navigation to the left to read about the available resources.
 ## Example Usage
 
 ```hcl
-# Configure the AWS Provider
```

Comment:
I believe this Terraform configuration for the provider is important here in some form, to show even a basic example of how it is done for newer practitioners.

I would suggest leaving a simple configuration without the credentials, such as the below:

```hcl
# Configure the AWS Provider
provider "aws" {
  region = "us-east-1"
}
```

Reply:
Mm, probably good to put a version constraint there too.

Reply:
Fixed in 71195a8d3619e13ec17a0059ed6714b7df852a0a with a version constraint.